### ISSUE TITLE
Add audio to video recordings

### ISSUE THREAD

**Issue by mrchantey** [cypress-io/cypress#17876]
### What would you like?

Option to record the audio output when running tests.

### Why is this needed?

Our app makes extensive use of the Web Audio API and, just as video recordings are helpful for UI testing, audio recordings would enhance our workflow dramatically.

**Comment by juanfonsecasolis** [comment]
Yes, I am needing this on my project. Please consider adding audio.

**Comment by acraileanu** [comment]
Is this being considered?

**Comment by nagash77** [comment]
@acraileanu this is not planned for development.  Any additional use cases and examples about why this would be valuable would be appreciated in case maybe there is a use case we are not considering.

**Comment by acraileanu** [comment]
@nagash77: this is useful for testing an application or a game that provides a rich experience that relies on sounds. An example is testing that sounds are played at a specific time (when an event occurs) and/or that they stop at a specific point in time / after an elapsed period. This gets a bit weirder when you add out-of-focus to the window (unexpected behaviors usually), so it is something you may want to look into. While it is good that you can programmatically test what happens, it is hard to review a failing test's video to understand what was going on - has the sound started playing / is it still playing / was it stopped?
